ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/TransactionScale.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTestSuite}
+import com.digitalasset.ledger.test_dev.Test.TextContainer
+import com.digitalasset.ledger.test_stable.Test._
```

Comment:
Perhaps explicit imports here?

```suggestion
import com.digitalasset.ledger.test_stable.Test.{Dummy, TextContainer}
```

(I highly suggest double-checking that this makes sense and will compile before committing it.)